app/lib/applied_standard_deduction.rb

```diff
@@ -0,0 +1,43 @@
+class AppliedStandardDeduction
+  attr_accessor :tax_year
+
+  def initialize(tax_return:)
+    @tax_return = tax_return
+    @tax_year = tax_return.year
+    @filing_status = tax_return.filing_status
+    @intake = tax_return.intake
+  end
+
+  def applied_standard_deduction
+    return nil if @intake.is_ctc? && @intake.home_location_puerto_rico?
+    raise NotImplementedError, "Standard deduction rule not implemented for #{@tax_year}" unless StandardDeductions.base_deductions(tax_year: @tax_year).present?
```

Comment:
Nit, but maybe we move the raise before the return nil? That would make the heirarchy of the method return from broad -> specific.